packages/fxa-content-server/app/scripts/lib/nimbus/index.ts

```diff
@@ -29,7 +29,7 @@ export default class Nimbus {
         return;
       }
 
-      this.experiments = await resp.json();
+      this.experiments = await resp.json().then((json) => json.Features);
```

Comment:
Yashika mentioned on Slack that we probably want the `Enrollments` object as well for metrics, so I'm going to take out this filter.